src/redux/middleware/ws/on_message.js

```diff
@@ -89,6 +90,16 @@ export default (alias, store) => (e = {}) => {
         break
       }
 
+      case 'auth.failure': { // WIP: waiting for a similar event to be introduced at the backend side
+        // if the authorisation token on hosted app is not valid, redirect to a page where it will be updated
+        if (!isElectronApp) {
+          const authURL = `${process.env.REACT_APP_UFX_API_URL}/honey`
```

Comment:
better to move this declaration in config.js, easy to track along with other envs